packages/opensnitch-ui-deb/opensnitch-ui-deb.pacscript

```diff
@@ -1,8 +1,9 @@
 name="opensnitch-ui-deb"
-version="1.5.0"
+version="1.5.1"
 url="https://github.com/evilsocket/opensnitch/releases/download/v${version}/python3-opensnitch-ui_${version}-1_all.deb"
+depends="qt-material"
```

Comment:
Why would this not be in the deb?

Reply:
![image](https://user-images.githubusercontent.com/70437021/164124173-da8292b4-8ee2-41d9-abff-5939aa838c6d.png)
it's not in the deb